packages/wp-story-editor/src/api/taxonomy.js

```diff
@@ -31,13 +35,24 @@ import apiFetch from '@wordpress/api-fetch';
  * @param {string} path API path.
  * @return {Promise} Taxonomies promise.
  */
-export function getTaxonomies(postType, path) {
-  return apiFetch({
+export async function getTaxonomies(postType, path) {
+  const result = apiFetch({
```

Comment:
```suggestion
  const result = await apiFetch({
```

Seems more readable to have the `await` here instead of doing `await result` below

Reply:
sounds good, fixed it.